services/utils/UtilsService.ts

```diff
@@ -157,19 +157,45 @@ export class UtilsService implements IUtilsService {
     }
 
     isPromise(obj: Object): boolean {
-        return obj && Object.prototype.toString.call(obj) === "[object Promise]";
+        return obj && Object.prototype.toString.call(obj) === '[object Promise]';
     }
 
     getQueryStringParams(query): {[key: string]: string} {
         let res = {};
-        (new URL(query)).searchParams.forEach((value: string, key: string,)=>{
+        (new URL(query)).searchParams.forEach((value: string, key: string,) => {
             res[key] = value;
         });
         return res;
     };
 
-    isUrl(str: string) : boolean {
+    isUrl(str: string): boolean {
         const regexp = /(ftp|http|https):\/\/(\w+:{0,1}\w*@)?(\S+)(:[0-9]+)?(\/|\/([\w#!:.?+=&%@!\-\/]))?/;
         return regexp.test(str);
     }
+
+
+    normalizeUrl({url, rootUrl}: NormalizeUrlParams) : string {
+
+        let normalizedUrl;
+
+        if (url.startsWith('http') || url.startsWith('http')) {
```

Comment:
This statement makes no sense. I think you can remove `|| url.startsWith('http')`?

Reply:
Probably a typo. Judging from the rest of the code, this was probably meant to be `url.startsWith('https')`

Reply:
**Yep!**